common/changes/@itwin/appui-layout-react/change_activeTab_in_preview_2023-11-24-13-38.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-layout-react",
+      "comment": "Added a preview feature - whenever a widget/tab is dropped somewhere, it becomes the active tab of the receiving container.",
```

Comment:
You can move this message to the "appui-react" one and leave this one empty, this package is internal and the behavior is exposed in AppUI-React anyway.